content/blog/2017-12-08-rprofile-jenny-bryan.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: rprofile-jenny-bryan
+title: ".rprofile: Jenny Bryan"
+date: 2017-11-10
```

Comment:
@kellobri can you fix this date to `2017-12-08` - that will make the preview work correctly